server/server.go

```diff
@@ -33,7 +34,9 @@ import (
 	confpb "google.golang.org/genproto/googleapis/api/expr/conformance/v1alpha1"
 	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 	rpc "google.golang.org/genproto/googleapis/rpc/status"
-	anypb "google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/anypb"
```

Comment:
I know that OSS go files don't complain about a missing alias, but internally the code wants these aliases, so if you wouldn't mind, please preserve them and use the designators `anypb`, `dpb`, `tpb`.  The `rpc` one actually needs to be changed to `rpcpb`